pkg/generate/code/check_test.go

```diff
@@ -74,6 +74,27 @@ func TestCheckRequiredFields_Attributes_StatusField(t *testing.T) {
 	)
 }
 
+func TestCheckRequiredFields_Attributes_StatusField_ReadMany(t *testing.T) {
```

Comment:
Please remove `_Attributes` from the test name (that refers to APIs that follow the Get Attributes pattern (i.e. SNS and SQS)